lib/super_digit.rb

```diff
@@ -1,15 +1,26 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(log10(n))
+# Space Complexity - O(log10(n))
 def super_digit(n)
-   
-end
+  if n < 10
+    return n
+  end
   
+  # return super_digit(sum_of_digits(n))
+  return super_digit((n % 10) + super_digit(n / 10))
```

Comment:
You're over calling super_digit here, as `n % 10` will always be less than 10, so `super_digit(n % 10)` will always return `n % 10`.